Makefile.Common

```diff
@@ -20,7 +20,7 @@ ALL_PKGS := $(shell go list $(sort $(dir $(ALL_SRC))) 2>/dev/null)
 # References: https://podman.io/getting-started/installation#build-tags and https://github.com/containers/podman/issues/6866
 GO_PODMAN_BUILD_TAGS=containers_image_openpgp,exclude_graphdriver_btrfs,exclude_graphdriver_devicemapper
 GO_BUILD_TAGS=$(GO_PODMAN_BUILD_TAGS)
-GOTEST_OPT?= -race -timeout 60s --tags=$(GO_BUILD_TAGS)
+GOTEST_OPT?= -race -timeout 300s --tags=$(GO_BUILD_TAGS)
```

Comment:
Is it worth making this an option that's configurable? It looks like the old `60s` is used in a couple of places (see line 24)

Reply:
it seems a bit overkill in my point of view. it is only a timeout and we need to give time to tests to end successfully in all cases so make it configurable does not seem very useful.

I rather seen this as a hard limit to prevent a too long fail answer for the user.
In my understanding, even with a very high value like 3600s this should not have any impact while tests should end before (and are run in parallel)

Reply:
👍